packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -249,6 +249,11 @@
     "secondaryText": "Please join this call using a compatible browser.",
     "moreHelpLink": "More help"
   },
+  "UnsupportedBrowserVersion": {
+    "primaryText": "Browser version not supported",
+    "secondaryText": "Please update your browser and then try joining the call again.",
+    "moreHelpLink": "See compatibility requirements"
```

Comment:
This default text suggests that the link will point to ACS help docs.

But there is no default implementation for `onTroubleshootingClick()` that does that.

I think we should addd default behavior for that callback so that the component works out of the box, and also so that this default text makes sense.

Reply:
So the goal here with this link is for contoso to provide their own page with this information since we don't want the component sending end users to the technical documentation for ACS. We don't provide a default behavior for the link and hide it should the callback not be provided.